README.md

```diff
@@ -1,2 +1,19 @@
 # code-review-bot
-code-review-bot
+
+Install Go Task
+
+```sh
+brew install go-task
+```
+
+Install dependencies
+
+```sh
+task install
+```
```

Comment:
Consider adding error handling suggestions for these commands in case they fail when run, which helps users troubleshoot issues more easily.